package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-dictionary",
-  "version": "3.9.0",
+  "version": "3.9.1",
```

Comment:
Can we remove this change? `standard-version` takes care of updating the version number

Reply:
Oh true, forgot about that